tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from zarr import MemoryStore
+
+
+@pytest.fixture
+def store(request: pytest.FixtureRequest) -> MemoryStore:
```

Comment:
```suggestion
def store(request: pytest.FixtureRequest) -> Store:
```
If you intend to return other stores in future?